signingscript/docker.d/init_worker.sh

```diff
@@ -205,6 +205,8 @@ case $ENV in
       mozillavpn)
         test_var_set 'AUTOGRAPH_AUTHENTICODE_PASSWORD'
         test_var_set 'AUTOGRAPH_AUTHENTICODE_USERNAME'
+        test_var_set 'AUTOGRAPH_MOZILLAVPN_PASSWORD'
+        test_var_set 'AUTOGRAPH_MOZILLAVPN_USERNAME'
```

Comment:
I found where this is set, trying to figure out how to request the creds.

Reply:
Anything public would go [here](https://github.com/mozilla-services/cloudops-infra/blob/master/projects/relengworker/k8s/values/signing.yaml), but nothing here is public, since we want to keep the user+pass together in sops.

We declare secrets [here](https://github.com/mozilla-services/cloudops-infra/blob/master/projects/relengworker/k8s/charts/signing/values.yaml#L35) but define them in k8s-sops. We then translate the secret from a yml entry into an env var [here](https://github.com/mozilla-services/cloudops-infra/blob/master/projects/relengworker/k8s/charts/signing/templates/secret.yaml#L12).

Reply:
If there are multiple pools that may reference some of these values, we tend to specify fake values in k8s-sops to avoid bustage related to missing secrets.

So it's possible that gecko signingscript pools may not refer to the mozillavpn values, but if adhoc-signing dep and dev refer to them, it may be best to specify the key/value pairs with a value of `"fake"`.

Reply:
Here's the cloudops-infra PR: https://github.com/mozilla-services/cloudops-infra/pull/3747